app/content/preload/content.js

```diff
@@ -12,4 +12,9 @@ specific language governing permissions and limitations under the License.
 
 import './hover-status';
 import './context-menu';
+import { readerify } from './reader';
+import Readability from './readability';
 import './scroll';
+
+window._readerify = readerify;
```

Comment:
We could do `readerify.bind(null, Readability)` or similar if we didn't want to have to attach window._Readability, but I'm not too worried about it either way at this point (maybe we'll want to instead fork Readability / remove unneeded code / expose our own readerify thing from that module)